rnc2rng/serializer.py

```diff
@@ -29,15 +31,21 @@ def __init__(self, indent=None):
 
     def reset(self):
         self.buf = []
-        self.needs = {}
         self.types = None
         self.ns = {}
+        self.typelibs = {}
         self.default = ''
         self.level = 0
 
     def write(self, s):
         self.buf.append(self.indent * self.level + s)
 
+    def datatypeLibrary(self, prefix):
```

Comment:
Please don't use camel-cased names for this, but translate it to `snake_case`.

Reply:
I used the name of the attribute it's calculating (i.e. the name set in the RELAX NG specification). But I can easily snake_case it.